### PR TITLE
RARO-364: fix incorrect use of Calendar

### PR DESCRIPTION
**Pull request by sjtower**

### CODE REVIEW

File: src/main/java/org/ambraproject/wombat/service/ArticleArchiveServiceImpl.java [hunk 1]
```diff
@@ -71,17 +70,20 @@
    * {@inheritDoc}
    */
   @Override
-  public Map<?, ?> getArticleDoisPerMonth(Site site, String year, String month) throws IOException {
-    SimpleDateFormat dateFormat = new SimpleDateFormat("yyyy-MM-dd");
-
+  public Map<?, ?> getArticleDoisPerMonth(Site site, String year, String month) throws IOException, ParseException {
     Calendar startDate = Calendar.getInstance();
-    startDate.set(Integer.parseInt(year), Arrays.asList(MONTHS).indexOf(month), 1);
```
Comment (sjtower) [inline src/main/java/org/ambraproject/wombat/service/ArticleArchiveServiceImpl.java]: This is the ultimate source of the bug. `Arrays.asList(MONTHS).indexOf(month)` was an incorrect use of this method, which expects EG Calendar.DECEMBER

Making a ticket to refactor this usage of `MONTHS` and `getMonthsForYear`. It's awkward and I think we can tighten it up.

**Comment by tpolischuk** [comment]
> @tpolischuk Do you know the status of this?

I honestly am not sure what this is regarding. The corresponding ticket https://jira.plos.org/jira/browse/RARO-364 was closed and appears to be about a different topic. It's also odd that it is a request to merge the PR directly to master. @anilop does this seem like something that needs to be picked back up?

**Comment by egh** [comment]
@tpolischuk This predates the move to gitflow, which explains the PR to master.

**Comment by anilop** [comment]
@egh, @tpolischuk - I am completely perplexed about this as well. It seems odd we would have let this lie for almost a year without a resolution, but clearly we did. his leads me to believe there were issues with this code and we didn't want to merge it.  I think we should close this code review.

**Comment by egh** [comment]
Thanks!

**Pull request closed without merging**